### DISCUSSION TITLE
Ability to choose which files to push to the Cloud from QField App

### DISCUSSION THREAD

**Discussion by cez2020** [opengisch/QField#4067] [Ideas]
Hello QFielders

In QGIS, I love having the ability to choose which files (layers) to update and push to the cloud for retrieval into the App. The App, however, doesn't offer that option. In my opinion, this capability would tremendously help those of us who have limited storage in our QField Cloud accounts by avoiding generating a copy of _every_ file every time a push is made from the App into the Cloud.

Ideas? Thoughts? Suggestions?

Thanks!

**Comment by suricactus** [comment]
Thank you for your input. We may consider this in the future, but it is not currently on our radar.
